test/entrypoint/srv_basic.lua

```diff
@@ -102,13 +102,21 @@ package.preload['mymodule-dependency'] = function()
 end
 
 package.preload['mymodule-permanent'] = function()
+    local log = require('log')
     return {
         role_name = 'myrole-permanent',
         permanent = true,
         get_role_name = function()
             return 'myrole-permanent'
         end,
 
+        init = function(opts)
+            log.info('--- init({is_master = %s})', opts.is_master)
```

Comment:
you could also add instance uuid or uri here

Reply:
They're already logged by luatest